genco-macros/src/quote.rs

```diff
@@ -244,6 +244,23 @@ impl<'a> Quote<'a> {
         Ok((req, Ast::Match { condition, arms }))
     }
 
+    fn parse_let(&self, input: ParseStream) -> Result<(Requirements, Ast)> {
+        input.parse::<Token![let]>()?;
+
+        let req = Requirements::default();
```

Comment:
It doesn't seem like `req` is used.

Reply:
I added it so that the method declaration is consistent with other methods, even if it's not useful here. Should I rather remove it?

Reply:
You can remove it if you want to. Your choice! I'll merge this later regardless.

Reply:
Let's keep it then, it makes my OCD happy 🤣